pyproject.toml

```diff
@@ -0,0 +1,37 @@
+[tool.poetry]
+name = "sceptre-hook-template"
+version = "0.0.1"
+packages = [{ include = "hook" }]
+readme = "README.md"
+homepage = "https://github.com/Sceptre/sceptre-hook-template"
+repository = "https://github.com/Sceptre/sceptre-hook-template"
+authors = ["Sceptre <sceptreorg@gmail.com>"]
+description = "A Github template for Sceptre hooks"
+keywords = ["sceptre", "sceptre-hook"]
+license = "Apache-2.0"
+classifiers = [
+  "Intended Audience :: Developers",
+  "Natural Language :: English",
+  "Environment :: Console",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11"
+]
+
+[tool.poetry.plugins."sceptre.hooks"]
```

Comment:
Why is this quoted like this?

Reply:
that's how you setup entrypoints using poetry, from the [poetry docs](https://python-poetry.org/docs/pyproject/#plugins)